doc/dev/installation.rst

```diff
@@ -81,6 +81,21 @@ The essential steps are:
 
    .. group-tab:: Linux Debian/Ubuntu
 
+      .. warning::
+
+        If using Anaconda or Miniconda, please make sure to upgrade ``libstdcxx-ng``:
+
+        .. code-block:: console
+
+          conda install -c conda-forge libstdcxx-ng
+
+        If not, you may receive the following error when running ``make test`` because the conda
+        environment is using old versions of ``libstdcxx-ng``.
+
+        .. code-block:: console
+
+          'GLIBCXX_3.4.x' not found
```

Comment:
@tzunghanjuang can you please move this below `.. code-block::` :) Once done I will approve

Reply:
Oh I suggested putting these up front because otherwise the last command in the block will fail

Reply:
oh, I think we gave opposite feedback at different times lol